src/CONST.ts

```diff
@@ -243,6 +243,10 @@ const CONST = {
         OLD_DOT_ANDROID: 'https://play.google.com/store/apps/details?id=org.me.mobiexpensifyg&hl=en_US&pli=1',
         OLD_DOT_IOS: 'https://apps.apple.com/us/app/expensify-expense-tracker/id471713959',
     },
+    ONYX_FETCH_STATUS: {
+        LOADING: 'loading',
+        LOADED: 'loaded',
```

Comment:
@fabioh8010 @blazejkustra thoughts on introducing these consts for the onyx loading?

Reply:
As explained [here](https://github.com/Expensify/App/pull/47310/files#r1718240603) we don't need to declare these values for now. If we need at some point in the future they should be declared inside `react-native-onyx`.